.gitattributes

```diff
@@ -0,0 +1,20 @@
+src/images/speakerhug_assets_2 filter=lfs diff=lfs merge=lfs -text
```

Comment:
so i think this isn't tracking all of them. you can do like `*.jpg`, `*.png` and `*.svg` and it will auto track everything we want it to